front/src/common/Pathfinding/TypeAndPath.tsx

```diff
@@ -137,6 +172,22 @@ export default function TypeAndPath({ zoomToFeature }: PathfindingProps) {
     }
   }
 
+  const onResultClick = (result: SearchResultItemOperationalPoint) => {
+    const newText = inputText.replace(searchState, result.trigram);
+
+    setInputText(newText);
+    setSearch('');
+    if (inputRef.current) inputRef.current.focus();
+  };
+
+  useEffect(() => {
+    if (debouncedSearchTerm) {
+      searchOperationalPoints();
+    } else if (searchResults.length !== 0) {
```

Comment:
if lenght can't be negative so you can just use `searchResults.length` in the condition

Reply:
done